source/manual/alerts/recreate-a-mongo-instance-in-aws.html.md

```diff
@@ -1,7 +1,7 @@
 ---
 owner_slack: "#2ndline"
 title: Re-create an AWS mongo instance
-section: Development VM
+section: Icinga Alerts
```

Comment:
I think the sections are case-sensitive, so this might need to be "Icinga alerts".